wallet/wallet.go

```diff
@@ -3691,7 +3691,14 @@ func (w *Wallet) ResetLockedOutpoints() {
 // LockedOutpoints returns a slice of currently locked outpoints.  This is
 // intended to be used by marshaling the result as a JSON array for
 // listlockunspent RPC results.
-func (w *Wallet) LockedOutpoints() []dcrdtypes.TransactionInput {
+func (w *Wallet) LockedOutpoints(ctx context.Context, accountName string) ([]dcrdtypes.TransactionInput, error) {
+	if accountName == "" || accountName == "*" {
+		return w.allLockedOutpoints(), nil
+	}
+	return w.acctLockedOutpoints(ctx, accountName)
+}
+
+func (w *Wallet) allLockedOutpoints() []dcrdtypes.TransactionInput {
```

Comment:
`listlockunspent` was never setting `tree` or `amount`.

In the case of `tree`, it was incorrectly always showing "0" for all outputs rather than omitting the field.  This should be fixed in both `allLockedOutpoints` and `acctLockedOutpoints`, and it's trivial to do with `op.Tree`.

For `amount`, the `wire.Outpoints` in `w.lockedOutpoints` do not have this data, so it's not trivial in `allLockedOutpoints`.  On the other hand, in `acctLockedOutpoints` you already have the msgTx, so that's simple.